test/helpers/environment.go

```diff
@@ -9,27 +9,12 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"k8s.io/klog/v2"
-	"k8s.io/klog/v2/textlogger"
-	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
 var TestEnv *TestEnvironment
 
 // Initialize the test environment. BeforeSuite will be only executed if this package is loaded by the test.
 var _ = BeforeSuite(func(ctx SpecContext) {
-	By("Initialize loggers for testing")
-	// Uninitialized logger spits stacktrace as warning during test execution
```

Comment:
I removed this. because this changes are unrelated. I need more testing to add this. I will file separate PR later.